src/Entity/Storage/AppStorage.php

```diff
@@ -94,6 +94,31 @@ public function loadUnchanged($id) {
     return parent::loadUnchanged($id);
   }
 
+  /**
+   * Load app by UUID.
+   *
+   * This function is more efficient than loadUnchanged(),
+   * because it does not need to cover the case
+   * when loading is done by App name.
+   *
+   * @param string $uuid
+   *   App UUID.
+   *
+   * @return \Drupal\apigee_edge\Entity\AppInterface|null
+   *   The unchanged entity, or NULL if the entity cannot be loaded.
+   *
+   * @TODO: this method should be also available in the AppStorageInterface,
+   *   but that would be a breaking change,
```

Comment:
Every line should be 80 characters long max, you can still add some text to this one. Please fix it.

Reply:
fixed